Modules/_sqlite/connection.c

```diff
@@ -913,30 +955,20 @@ static int _authorizer_callback(void* user_arg, int action, const char* arg1, co
     PyObject *ret;
     int rc;
```

Comment:
Suggestion: Be defensive and initialize this to `SQLITE_DENY`?

Reply:
I agree.